src/components/Main/Logout.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import useHttpRequest from '../../hooks/useHttpRequest';
+import useCookie from '../../hooks/useCookie';
+
+const Logout = () => {
```

Comment:
I don't think that we need logout component. It's enough to have just logout button in header

Reply:
Fixed.